### PR TITLE
4.11 Release Notes updates for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s) : 4.11

Issue : https://issues.redhat.com/browse/MULTIARCH-2570

Link to docs preview :
For P : http://file.del.redhat.com/~aprabhu/411-release-notes/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

Reviewers :
For P : Manoj Kumar, Thomas L Falcon

Additional information : 
- This PR is raised to update 4.11 RN with a known issue seen on Power platform.
- BZ : https://bugzilla.redhat.com/show_bug.cgi?id=2107674

Branch : enterprise-4.11
@SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1934,6 +1934,20 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
 * There is currently a known issue in the *Add Helm Chart Repositories* form to extend the Developer Catalog of a project. The *Quick Start* guides shows that you can add the `ProjectHelmChartRepository` CR in the desired namespace whereas it does not mention that to perform this you need permission from the kubeadmin. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054197[BZ#2054197])
 * There is currently a known issue. If you create an instance of `ProjectHelmChartRepository` custom resource (CR) that uses TLS verification, you cannot list the repositories and perform Helm-related operations. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
 
+* When running {product-title} on bare-metal IBM Power, there is a known issue that the Petitboot bootloader is unable to populate boot configurations for some {op-system} live images. In such cases, after booting nodes with PXE to install {op-system}, the expected live image disk configuration might not be visible upon reboot.
```
Comment (mjpytlak) [inline release_notes/ocp-4-11-release-notes.adoc:1937]:
```suggestion
* When running {product-title} on bare-metal IBM Power, there is a known issue that the Petitboot bootloader is unable to populate boot configurations for some {op-system} live images. In such cases, after booting nodes with PXE to install {op-system}, the expected live image disk configuration might not be visible.
```
I believe `booting nodes` implies the reboot. Purely a suggestion. Not sure if removing `upon reboot` changes the technical accuracy.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1934,6 +1934,20 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
...
++
+As a workaround, you can boot manually with `kexec` from the Petitboot shell.
++
+Identify the disk that holds the live image, in this example `nvme0n1p3` and run the following commands:
```
Comment (mjpytlak) [inline release_notes/ocp-4-11-release-notes.adoc:1941]:
```suggestion
Identify the disk that holds the live image, in this example `nvme0n1p3`, and run the following commands:
```

**Review by mjpytlak — COMMENTED** [review]
@alishaIBM Few small suggestions.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -2032,6 +2032,20 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
...
++
+Identify the disk that holds the live image, in this example `nvme0n1p3`, and run the following commands:
++
+[source,bash]
+----
+# cd /var/petitboot/mnt/dev/nvme0n1p3/ostree/rhcos-*/
+# kexec -l vmlinuz-*.ppc64le -i initramfs-*.img -c "ignition.firstboot rd.neednet=1 ip=dhcp $(grep options /var/petitboot/mnt/dev/nvme0n1p3/loader/entries/ostree-1-rhcos.conf | sed 's,^options ,,')" && kexec -e
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2107674[BZ#2107674])
```
Comment (mjpytlak) [inline release_notes/ocp-4-11-release-notes.adoc:2047]:
```suggestion
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2107674[*BZ#2107674]*)
```
Apologies @alishaIBM. I didn't catch this first time through. The BZ link needs to be bold.

**Review by mjpytlak — COMMENTED** [review]
@alishaIBM Apologies. One more formatting item to address.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 3 above)
Comment (alishaIBM) [reply release_notes/ocp-4-11-release-notes.adoc:2047]: Thanks @mjpytlak for the correction. 
As discussed [over slack conversation](https://coreos.slack.com/archives/C025XCRRH28/p1659020167361189?thread_ts=1659005775.250879&cid=C025XCRRH28) this change will be handled by the RH docs team.

Marking the comment as resolved.

**Pull request merged**